cfme/tests/automate/custom_button/test_container_objects.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+import fauxfactory
+
+from widgetastic_patternfly import Dropdown
+
+from cfme.containers.provider import ContainersProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.log import logger
+
+
+pytestmark = [
+    pytest.mark.tier(2),
+    pytest.mark.usefixtures("setup_provider"),
+    pytest.mark.provider([ContainersProvider], selector=ONE_PER_TYPE),
+]
+
+CONTAINER_OBJECTS = [
```

Comment:
**Required** This should a list of the collection classes, ie:
```
CONTAINER_COLLECTIONS = [
"container_images",
"container_nodes",
"container_pods",
"container_projects",
"container_templates",
"container_volumes",
]``` 

If we do this, we can use it to call the collection classes directly.